examples/example-xds/build.gradle

```diff
@@ -1,5 +1,9 @@
 plugins {
     id 'application' // Provide convenience executables for trying out the examples.
+    // ASSUMES GRADLE 5.6 OR HIGHER. Use plugin version 0.8.10 with earlier gradle versions
+    id 'com.google.protobuf' version '0.8.10'
```

Comment:
Use 0.8.13 please, make it consistent with other examples so the person that upgrades the version will not miss thi one.